src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -2663,6 +2663,34 @@ void M()
 }"); // No quick info (see issue #16667)
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task TestLambdaDiscardParameter_FirstDiscard()
+        {
+            await TestAsync(
+@"class C
+{
+    void M()
+    {
+        System.Func<string, int, int> f = ($$_, _) => 1;
+    }
+}",
+                MainDescription($"({FeaturesResources.parameter}) string "));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task TestLambdaDiscardParameter_SecondDiscard()
+        {
+            await TestAsync(
+@"class C
+{
+    void M()
+    {
+        System.Func<string, int, int> f = (_, $$_) => 1;
+    }
+}",
+                MainDescription($"({FeaturesResources.parameter}) int "));
```

Comment:
seems odd.  i feel like we should have special quick-info langauge for a discard.

Reply:
So, other discards like `_ = 1;` are displayed `int _` in QuickInfo. I could do the same here.
But I agree with you that it probably would be better to display all discards like `int (discard)` or something, so we don't give the impression that there is a variable named `_`.

Reply:
I'll change parameter discards to display the same as other discards (ie. with a `_` rather than a blank).

Reply:
Modified QuickInfo to display `(discard) int _`.